web/static.go

```diff
@@ -28,7 +27,7 @@ func (w *Web) InitStatic() {
 		}
 
 		staticDir, _ := fileutils.FindDir(model.CLIENT_DIR)
-		mlog.Debug(fmt.Sprintf("Using client directory at %v", staticDir))
+		mlog.Debug("Using client directory at", mlog.String("staticDir", staticDir))
```

Comment:
I think here the debug string could simply be `Using client directory.` since the `at` word will be missing context.
Also `"staticDir"` could probably become `"clientDir"`.

Reply:
done